src/extension.ts

```diff
@@ -874,12 +881,20 @@ export async function activate(context: vscode.ExtensionContext) {
 						);
 					}
 					newPrivateCodemodNames.push(codemodHash);
-					await writeFile(
-						privateCodemodNamesPath,
-						JSON.stringify({
-							names: newPrivateCodemodNames,
-						}),
-					);
+					await Promise.all([
```

Comment:
why do we need Promise all here?

Reply:
I think it's negligent with two promises, it's easier to read it when it's one by one (but it's ok to leave it as it is)